.ado/azure-pipelines.publish.yml

```diff
@@ -81,7 +81,7 @@ extends:
                 displayName: 'yarn install'
 
               - script: |
-                  yarn buildci
+                  yarn buildci [test]
```

Comment:
This doesn't get in the way of running the test? It's passing "[test]" as an argument now

Reply:
Yes, my bad. fixed.